### PR TITLE
#5 - Keycode first draft of html/.js

### PR DESCRIPTION
**Pull request by ruarplum**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+// const eventKeyResult = document.querySelector(".key-result");
+// const eventCodeResult = document.querySelector(".code-result");
+// const eventWhichResult = document.querySelector(".which-result");
+
+document.addEventListener("keydown", (event) => {
+    console.log(event);
+    firstPage.style.display = "none";
+    
+    let eventKeyResult = event.key;
+    console.log(eventKeyResult);
```
Comment (tamjankowska) [inline app.js:14]: Do you still need these console.logs in place? Or is this leftover test code?

File: app.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    let eventCodeResult = event.code;
+    console.log(eventCodeResult);
+
+    let eventWhichResult = event.which;
+    console.log(eventWhichResult)
+
+    let paraKey = document.querySelector(".keyResult");
+    paraKey.textContent = eventKeyResult;
+
```
Comment (tamjankowska) [inline app.js:24]: Do you prefer storing the event properties in variables rather than just passing them in directly to textContent?

**Review by tamjankowska — APPROVED** [review]

File: app.js (hunk 2 above)
Comment (ruarplum) [reply app.js:24]: so I could do:

.keyResult.textContent = eventKeyResult;??

File: app.js (hunk 1 above)
Comment (ruarplum) [reply app.js:14]: will comment these out

File: app.js (hunk 2 above)
Comment (tamjankowska) [reply app.js:24]: I guess I meant more along the lines of `paraWhich.textContent = event.which` or even `document.querySelector(".whichResult").textContent = event.which`

**Pull request merged**